satellite/peer.go

```diff
@@ -249,7 +250,13 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config, ve
 	{ // setup listener and server
 		log.Debug("Starting listener and server")
 		sc := config.Server
-		options, err := tlsopts.NewOptions(peer.Identity, sc.Config)
+
+		revDB, err := revocation.NewDBFromCfg(sc.Config)
```

Comment:
databases are passed in as an argument to avoid logic from depending on specific implementations... revocation was one of those exceptions that we hadn't yet fixed

Reply:
Can you clarify what you mean here? Is there a change you'd like to see made?